exercises/practice/word-search/test/word_search_test.exs

```diff
@@ -0,0 +1,463 @@
+defmodule WordSearchTest do
+  use ExUnit.Case
+  alias WordSearch.Location
+
+  # Grid rows and columns are 1-indexed.
+  # @tag :pending
+  test "Should accept an initial game grid and a target search word" do
+    grid = "jefblpepre"
+    words = ["clojure"]
+    output = WordSearch.search(grid, words)
+    expected = %{"clojure" => nil}
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate one word written left to right" do
+    grid = "clojurermt"
+    words = ["clojure"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "clojure" => %Location{from: {1, 1}, to: {1, 7}}
+    }
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate the same word written left to right in a different position" do
+    grid = "mtclojurer"
+    words = ["clojure"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "clojure" => %Location{from: {1, 3}, to: {1, 9}}
+    }
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate a different left to right word" do
+    grid = "coffeelplx"
+    words = ["coffee"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "coffee" => %Location{from: {1, 1}, to: {1, 6}}
+    }
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate that different left to right word in a different position" do
+    grid = "xcoffeezlp"
+    words = ["coffee"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "coffee" => %Location{from: {1, 2}, to: {1, 7}}
+    }
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate a left to right word in two line grid" do
+    grid = """
+    jefblpepre
+    tclojurerm
+    """
+
+    words = ["clojure"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "clojure" => %Location{from: {2, 2}, to: {2, 8}}
+    }
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate a left to right word in three line grid" do
+    grid = """
+    camdcimgtc
+    jefblpepre
+    clojurermt
+    """
+
+    words = ["clojure"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "clojure" => %Location{from: {3, 1}, to: {3, 7}}
+    }
+
+    assert output == expected
+  end
+
+  @tag :pending
+  test "Should locate a left to right word in ten line grid" do
+    grid = """
+    jefblpepre
+    camdcimgtc
+    oivokprjsm
+    pbwasqroua
+    rixilelhrs
+    wolcqlirpc
+    screeaumgr
+    alxhpburyi
+    jalaycalmp
+    clojurermt
+    """
+
+    words = ["clojure"]
+    output = WordSearch.search(grid, words)
+
+    expected = %{
+      "clojure" => %Location{from: {10, 1}, to: {10, 7}}
```

Comment:
I wonder if we should make the coordinates more explicit like this:
```suggestion
      "clojure" => %Location{from: %{row: 10, column: 1}, to: %{row: 10, column: 7}}
```

When I first saw `{10, 1}`, I thought: `{x, y}`, which is basically the opposite of what it really is 😁

Reply:
☝🏻 agreed, it isn't intuitive from a pair of numbers to know what they mean if not `{x, y}`

Reply:
That's quite interesting, to me `{row, column}` is by far the most intuitive. I guess it's because I'm from a physics/math background and that's how you manipulate matrices. 
Originally the tests included explicitly row and column, I'll put them back in. Sorry about that!